### PR TITLE
Remove local Slice

### PR DESCRIPTION
**Pull request by bernardnormier**
This PR removes local Slice for all language mappings. This includes remove "local" from the grammar.

### CODE REVIEW

**Review by pepone — APPROVED** [review]
Looks good!

File: cpp/src/Slice/JavaUtil.cpp [hunk 1]
```diff
@@ -459,15 +459,7 @@ class MetaDataVisitor : public ParserVisitor
             }
             else if(i->find("delegate") == 0)
```
[the remark this answers is not in the document]
Comment (bernardnormier) [reply cpp/src/Slice/JavaUtil.cpp:460]: We will need additional cleanup in follow-up PRs. I didn't remove all the local-only metadata. "delegate" is local-only:
https://doc.zeroc.com/ice/3.7/the-slice-language/slice-metadata-directives#id-.SliceMetadataDirectivesv3.7-delegate

so is async-oneway, cpp:noexcept, js:async, swift:noexcept and more.

File: cpp/src/Slice/msbuild/slice.vcxproj [hunk 2]
```diff
@@ -118,4 +122,4 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets" />
-</Project>
+</Project>
```
Comment (InsertCreativityHere) [inline cpp/src/Slice/msbuild/slice.vcxproj:125]: Looks like the newline at the end of this file got eaten.

**Review by InsertCreativityHere — COMMENTED** [review]
Probably won't finish reviewing this until tonight.
I've looked through most of the parser stuff and it looks good,
so feel free to merge, and I'll just comment on the PR afterwards.

**Pull request merged**